pkg/kritis/review/review.go

```diff
@@ -104,10 +104,14 @@ func (r Reviewer) ReviewISP(images []string, isps []v1beta1.ImageSecurityPolicy,
 	for _, isp := range isps {
 		glog.Infof("Validating against ImageSecurityPolicy %s", isp.Name)
 		// Get all AttestationAuthorities in this policy.
-		auths, err := r.getAttestationAuthoritiesForISP(isp)
+		a, err := r.getAttestationAuthorityForISP(isp)
 		if err != nil {
 			return err
 		}
+		auths := make([]v1beta1.AttestationAuthority, 0)
+		if a != nil {
+			auths = append(auths, *a)
+		}
 		for _, image := range images {
 			glog.Infof("Check if %s as valid Attestations.", image)
 			notAttestedBy := r.findUnsatisfiedAuths(image, auths)
```

Comment:
I suggest factoring out this logic into a helper:
func (r Reviewer) isAttestedBy(image string, auth AttestationAuthority) (bool, error)
or something like it